extensions/redcore.xml

```diff
@@ -7,7 +7,7 @@
     <authorUrl>redweb.dk</authorUrl>
     <copyright>Copyright (C) 2008 - 2020 redWEB.dk. All rights reserved.</copyright>
     <license>GNU General Public License version 2 or later, see LICENSE.</license>
-    <version>2.1.1</version>
+    <version>2.1.2</version>
     <releaseName>beta1</releaseName>
```

Comment:
Can you please use v.2.1.1 beta2 instead?

Reply:
done